lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1879,6 +1879,46 @@ def text(self, x, y, z, s, zdir=None, **kwargs):
     text3D = text
     text2D = Axes.text
 
+    def _broadcast_with_masks(self, inputs, compress=False):
```

Comment:
This looks like a method that might fit in to `cbook.py`...

The closest that is present there is `_combine_masks`, but that seems to only handle the 1D masked array case.

I also think this is perhaps better as a pure function rather than an instance method, as it has nothing actually to do with `self`.

For consistency with `cbook`, I may also suggest passing inputs as `*args` rather than a single arg that is a list.

Reply:
That all makes sense to me @ksunden, moved it over to cbook.